SonarQube.TeamBuild.PreProcessor/AnalysisConfigGenerator.cs

```diff
@@ -0,0 +1,80 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="AnalysisConfigGenerator.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
```

Comment:
All of the code in this class was extracted from the preprocessor class. Currently it behaves exactly the same.